DEVELOPMENT-GUIDE.md

```diff
@@ -0,0 +1,85 @@
+# ACCP Development Guidance
+Correctness and code-safety is paramount in the Amazon Corretto Crypto Provider.
+
+The purpose of this guide is not to provide all information needed for development within ACCP.
```

Comment:
s/within/with

Reply:
Nope. This doesn't teach people how to use ACCP (and thus "with it") but how to work "within" it and modify its innards. I'll still reword it though.